### PR TITLE
Add eth top transactions metric

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: src/functions/blockchainFunctions.js [hunk 1]
```diff
@@ -160,3 +160,30 @@ function historicalBalance_ (slug, from, to, address) {
...
   }))
 }
+
+function ethTopTransactions_ (slug, from, to, limit, transactionType) {
+  var results = getApiClient_().fetchEthTopTransactions(slug, from, to, limit, transactionType)
+  assertHasData_(results)
+
+  var headers = [
+    'Date',
+    'From Address',
```
Comment (IvanIvanoff) [inline src/functions/blockchainFunctions.js]: Shouldn't these names be concatenated? `FromAddress`, `FromAddressesIsExchange`, etc.?

Comment (spiderjako) [reply src/functions/blockchainFunctions.js]: I don't think so at least. The other functions are made with spaces, I took them as a model to make these. If you say so I will make them camelCase

File: src/functions/blockchainFunctions.js [hunk 2]
```diff
@@ -160,3 +160,30 @@ function historicalBalance_ (slug, from, to, address) {
     ]
   }))
 }
+
+function ethTopTransactions_ (slug, from, to, limit, transactionType) {
+  var results = getApiClient_().fetchEthTopTransactions(slug, from, to, limit, transactionType)
```
Comment (kalkov) [inline src/functions/blockchainFunctions.js]: Should we have `ALL` for default transactionType here? 
@IvanIvanoff @tspenov

File: src/functions/blockchainFunctions.js [hunk 3]
```diff
@@ -160,3 +160,30 @@ function historicalBalance_ (slug, from, to, address) {
...
+
+  return [headers].concat(results.ethTopTransactions.map(function (result) {
+    return [
+      formatDatetimeField_(result.datetime),
+      result.fromAddress.address,
+      result.fromAddress.isExchange,
+      result.toAddress.address,
+      result.toAddress.isExchange,
+      result.trxHash,
+      result.trxValue
```
Comment (kalkov) [inline src/functions/blockchainFunctions.js]: `formatNumber_`

File: test/setup.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ const historicDataTo = subDays(endOfYesterday(), 200)
 const historicDataFrom = startOfDay(subDays(historicDataTo, 205))
 
 const ethAddress = '0x1f3df0b8390bb8e9e322972c5e75583e87608ec2'
+const transactionType = 'ALL'
```
Comment (kalkov) [inline test/setup.js]: Not needed here

Comment (spiderjako) [reply test/setup.js]: You're right, I put it in the test for the function and removed it from setup.js

File: src/functions/blockchainFunctions.js (hunk 3 above)
Comment (spiderjako) [reply src/functions/blockchainFunctions.js]: My bad, already fixed

**Pull request merged**